signalfx-org-metrics/README.md.jinja

```diff
@@ -11,43 +11,72 @@
+-  Cloud service integration metrics that measure how Infrastructure Monitoring pulls data from cloud services such as Amazon Web Services (**AWS**) or Google Cloud Platform (**GCP**), such as the number of calls to each GCP Stackdriver client method, or the number of the calls to the AWS CloudWatch API that AWS has throttled
 
-Your organization is not charged for these metrics, and they do not count against any limits.
+-  Metrics that measure your use of resources for which you can specify limits, such as the number of custom MTS you've created. These metrics
+   help you track usage and avoid going over your organization limits.
 
+You're not charged for these metrics, and they do not count against any limits.
 
-## USAGE
 
-Admins can see some of these values in built-in charts on the Organization Overview page. And, like any other metrics, you can display them on charts you build to monitor your system health.
+## USAGE
 
+Admins can view some of these metrics in built-in charts on the Organization Overview page. Like other metrics, you can display them in custom charts you create in order to monitor your system health.
 
-### About ByToken metrics
+### Metric pairs that provide total and ByToken values
 
 Some metrics send both a total value and a ByToken value, such as `sf.org.numAddDatapointCalls` and `sf.org.numAddDatapointCallsByToken`.
 
-The sum of all the ByToken values may be less than the value of the counterpart (non-token-based) metric; this is because no per-token value is sent for data that SignalFx retrieves on your behalf via AWS CloudWatch, GCP StackDriver, AppDynamics, or New Relic, because data sent from those integrations is not associated with a token. For example, if you sum the values sent for `sf.org.numAddDatapointCallsByToken`, the value may be less than the value of `sf.org.numAddDatapointCalls`, because the latter includes data from the specified integrations while the former does not.
+The sum of all the ByToken values for a metric may be less than the value of its non-token-based counterpart metric.
+The sums differ because Infrastructure Monitoring doesn't use a token to retrieve cloud service data you've requested with an integration.
+This applies to AWS CloudWatch, GCP StackDriver, AppDynamics, and New Relic.
+
+For example, if you sum the values for `sf.org.numAddDatapointCallsByToken`, the result may be less than the value of
+`sf.org.numAddDatapointCalls`, because `sf.org.numAddDatapointCalls` includes data from cloud service integrations,
+while `sf.org.numAddDatapointCallsByToken` does not.
+
+### Metrics with values for each metric type
+
+Some metrics have a value for each metric type (counter, cumulative counter or gauge), so you have three MTS per metric. Each MTS has a dimension named `category` with a value of `COUNTER`, `CUMULATIVE_COUNTER`, or `GAUGE`. Because you can have multiple MTS for these metrics, you need to use the `sum()` SignalFlow function to see the total value.
```

Comment:
```suggestion
Some metrics have a value for each metric type (counter, cumulative counter, or gauge), so you have three MTS per metric. Each MTS has a dimension named `category` with a value of `COUNTER`, `CUMULATIVE_COUNTER`, or `GAUGE`. Because you can have multiple MTS for these metrics, you need to use the `sum()` SignalFlow function to see the total value.
```

Reply:
Fixed.